compiler/rustc_middle/src/mir/interpret/mod.rs

```diff
@@ -125,7 +125,9 @@ pub use self::error::{
 
 pub use self::value::{get_slice_bytes, ConstAlloc, ConstValue, Scalar, ScalarMaybeUninit};
 
-pub use self::allocation::{alloc_range, AllocRange, Allocation, InitMask, Relocations};
+pub use self::allocation::{
+    alloc_range, AllocError, AllocRange, Allocation, InitMask, Relocations,
```

Comment:
Not that I have any objections, but why is this re-exported in this PR? It does not seem to be used.

Reply:
This is left-over from a previous local modification. I'll remove it.

Reply:
Done.